patterns/2-structured/core-team.md

```diff
@@ -0,0 +1,102 @@
+## Title
+
+Core Team
+
+## Patlet
+
+Is your InnerSource project difficult for people to work with, yet no one is interested in improving it?
+Establish a core team to take care of the project's fundamental items so that contributors are enabled to add and use the features that provide value to their scenarios.
```

Comment:
It's not really emphasized here, but is this a somehow-dedicated core team? Or at least a team whose members have a prioritized emphasis on this IS project's technical health and maintenance? Doesn't need to be necessarily answered here, but probably should be explored a little in the solution part of the write-up.

Reply:
Updated to include the work `dedicated` in introducing the core team.